elfi/methods/methods.py

```diff
@@ -788,7 +791,8 @@ def __init__(self, model, target=None, outputs=None, batch_size=1,
         if initial_evidence % self.batch_size != 0:
             raise ValueError('Initial evidence must be divisible by the batch size')
 
-        self.acquisition_method = acquisition_method or LCBSC(target_model)
+        priors = [self.model[p] for p in self.parameters]
+        self.acquisition_method = acquisition_method or LCBSC(target_model, priors=priors)
```

Comment:
We could add the noise here too, it would be beneficial in the parallel execution.